frontend/src/i18n/de-DE/index.js

```diff
@@ -0,0 +1,353 @@
+    completed: 'Vollständig',
+    type: 'Type',
+    description: 'Beschreibung',
+    observation: 'Beobachtung',
+    references: 'Referenzen (eine pro Zeile)',
+    customFields: 'Benutzerdefinierte Felder',
+    affectedAssets: 'Betroffene Assets',
+    courseOfActions: 'Vorgehensweise',
+    template: 'Vorlage',
+    client: 'Kunde',
+    reviewers: 'Prüfer',
+    collaborator: 'Mitarbeiter',
+    collaborators: 'Mitarbeiter',
+    startDate: 'Startdatum',
+    endDate: 'Enddatum',
+    reportingDate: 'Berichtsdatum',
+    auditScope: 'Geltungsbereich (einer pro Zeile)',
+    import: 'Import',
+    export: 'Export',
+    hostsAssociateScopes: 'Importierte Hosts mit Geltungsbereichen assoziieren',
+    handleCustomData: 'Handle Custom Data',
+    companies: 'Firmen',
+    clients: 'Kunden',
+    templates: 'Vorlagen',
+    addClient: 'Kunde hinzufügen',
+    editClient: 'Kunde bearbeiten',
+    firstname: 'Vorname',
+    lastname: 'Nachname',
+    email: 'E-Mail',
+    function: 'Funktion',
+    phone: 'Telefon',
+    cell: 'Mobiltelefon',
+    role: 'Rolle',
+    collatorator: 'Mitarbeiter',
+    addCollaborator: 'Mitarbeiter hinzufügen',
+    editCollaborator: 'Mitarbeiter bearbeiten',
+    addCompany: 'Firma hinzufügen',
+    editCompany: 'Firma bearbeiten',
+    logo: 'Logo',
+    quantifier: '',
+    languages: 'Sprachen',
+    extension: 'Erweiterung',
+    createTemplate: 'Vorlage erstellen',
+    file: 'Datei',
+    editTemplate: 'Vorlage bearbeiten',
+    importVulnerabilities: 'Schwachstellen importieren',
+    importVulnerabilitiesInfo: `Schwachstellen aus .yml- oder .json-Datei importieren. (Serpico-Format wird akzeptiert)<br />
+    Dies erlaubt es Ihnen einfach eine Reihe von Schwachstellenvorlagen zu importieren.`,
+    importVulnerabilitiesOk: 'Alle <strong>{0}</strong> Schwachstellen wurden erstellt',
+    importVulnerabilitiesAllExists: 'Alle <strong>{0}</strong> Schwachstellentitel existieren bereits',
+    importVulnerabilitiesPartial: '<strong>{0}</strong> Schwachstellen wurden erstellt<br /><strong>{1}</strong> Schwachstellentitel existieren bereits',
+    exportVulnerabilities: 'Schwachstellen exportieren',
+    exportVulnerabilitiesInfo: `Exportiert Schwachstellen im yaml-Format. Dieser Export kann später einfach wieder zurück-importiert werden`,
+    deleteAllVulnerabilities: 'Alle Schwachstellen löschen',
+    deleteAllVulnerabilitiesInfo: `Löscht alle existierende Schwachstellen.<br />
+    <strong>Diese Aktion ist unwiderruflich!!</strong>`,
+    customSections: 'Eigene Abschnitte',
+    listOfLanguages: 'Liste der Sprachen',
+    editLanguages: 'Sprache bearbeiten',
+    locale: 'Locale',
+    auditTypesUsedInAudits: 'Audit-Typen die in Audits genutzt werden',
+    languageUsedInAuditsAndVuls: 'Sprachen die in Audits und Schwachstellen genutzt werden',
+    addSections: 'Abschnitt hinzufügen',
+    hideBuiltInSections: 'Vorgefertigte Abschnitte ausblenden',
+    listOfAuditTypes: 'List der Audit-Typen',
```

Comment:
Did go over those changes, looking good to me!
I think I found a typo here, should be `Liste der Audit-Typen`